examples/deployment/src/main/yaml/example-cluster-persistence.yaml

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at
+# http://oss.oracle.com/licenses/upl.
+#
+# Example 4
+# Enables active Persistence
+apiVersion: coherence.oracle.com/v1
+kind: CoherenceCluster
+metadata:
+  name: example-cluster
+spec:
+  imagePullSecrets:
+    - name: coherence-example-secret
+  jvm:
+    heapSize: 512m
+  ports:
+    - name: metrics
+      port: 9612
+  coherence:
+    logLevel: 5
+    image: iad.ocir.io/odx-stateservice/test/coherence:12.2.1.4.0-b74630
+    metrics:
+      enabled: true
+  application:
+    image: deployment-example:2.0.0
+    imagePullPolicy: IfNotPresent
+  # 
+  # Individual cluster roles
+  #
+  roles:
+    - role: storage
+      replicas: 3
+      coherence:
+        cacheConfig: storage-cache-config.xml
+        persistence:
+          enabled: true
+          persistentVolumeClaim:
+            accessModes:
+            - ReadWriteOnce
+            resources:
+              requests:
+                storage: 1Gi
+      application:
+        image: deployment-example:2.0.0
+        imagePullPolicy: IfNotPresent
```

Comment:
`IfNotPresent` is the Kubernetes default for `imagePullPolicy` so it isn't really required

Reply:
fixed